### PR TITLE
completed image,input and documentation

### PR DESCRIPTION
**Pull request by archit7448**
completed image ,input, button and documentation
https://agitated-hawking-b2d497.netlify.app/components/buttons/button

### CODE REVIEW

File: Components/Buttons/buttons.css [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
+    color: var(--color-secondary-white);
+    background-color: var(--color-primary-blue);
 }
 .button-primary:hover{
-    background-color: var(--color-primary-green-bright);
+    background-color: var(--color-neon-blue);
     transition: 0.1s;
 }
 .button-secondary{
     border: 1px solid black;
```
Comment (zoef-yt) [inline Components/Buttons/buttons.css]: use var colour

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: Components/Buttons/buttons.css [hunk 2]
```diff
@@ -5,35 +5,32 @@
...
+      color: var(--color-neon-blue);
       transition: 0.1s;
 }
 .button-link{
-    color: var(--color-primary-green);
+    color: var(--color-neon-blue);
 }
 .button-icon{
     font-size: 2rem;
     color: grey;
```
Comment (iRohitGaur) [inline Components/Buttons/buttons.css]: use var for color

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Comment by archit7448** [comment]
updated changes

**Pull request merged**